src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/ASwitchFlows.groovy

```diff
@@ -0,0 +1,39 @@
+package org.openkilda.functionaltests.helpers.model
+
+import org.openkilda.functionaltests.model.cleanup.CleanupManager
+import org.openkilda.testing.service.lockkeeper.LockKeeperService
+import org.openkilda.testing.service.lockkeeper.model.ASwitchFlow
+import org.springframework.beans.factory.annotation.Autowired
+
+import static org.openkilda.functionaltests.model.cleanup.CleanupActionType.RESTORE_A_SWITCH_FLOWS
+import static org.springframework.beans.factory.config.ConfigurableBeanFactory.SCOPE_PROTOTYPE
+
+import groovy.util.logging.Slf4j
+import org.springframework.context.annotation.Scope
+import org.springframework.stereotype.Component
+
+
+@Slf4j
+@Component
+@Scope(SCOPE_PROTOTYPE)
+class ASwitchFlows {
+    @Autowired
+    LockKeeperService lockKeeperService
+    @Autowired
+    CleanupManager cleanupManager
+
+    void addFlows(List<ASwitchFlow> flows) {
+        cleanupManager.addAction(RESTORE_A_SWITCH_FLOWS, {lockKeeperService.removeFlows(flows)})
```

Comment:
WDYT: do we need to add some clarification of our topology.isls recovering (without a-switch flow we can have some other isls that will be auto-discovered by Kilda)?

Reply:
At the moment, in practice we only restore a-switch-related ISLs at this step and these ISLs return into expected state while other post-test actions happen (probably due to DoCleanupListener). If it would start to break, then yes, we'll need to add here some action to wait for ISL to appear/disappear.
But this doesn't seem like a trivial task for me.